submissions/ArthurGorbenko/Memory Pair Game/js/script.js

```diff
@@ -0,0 +1,128 @@
+const valuesOfBackPos = [
+  "20px 0px",
+  "20px -180px",
+  "20px -350px",
+  "20px -525px",
+  "-190px 0px",
+  "-390px 0px",
+  "-600px 0px",
+  "-810px 0px"
+];
+
+let cards = document.getElementsByClassName("card");
+const container = document.querySelector(".container");
+
+for (let i = 0; i < cards.length; i++) {
+  addElements(cards[i], i);
+}
+
+function addElements(element, i) {
+  let flipper = document.createElement("div");
+  flipper.className = "flipper";
+  element.appendChild(flipper);
+
+  let front = document.createElement("div");
+  front.className = "front";
+  flipper.appendChild(front);
+
+  let back = document.createElement("div");
+  back.className = "back";
+  back.dataset.numberOfElement = i;
+  flipper.appendChild(back);
+}
+
+let backs = document.querySelectorAll(".back");
+
+generateBackGrn(backs, valuesOfBackPos);
+
+function generateBackGrn(elements, arrayOfBackGrnds) {
+  let arrayOfElements = Array.from(elements);
+  arrayOfElements.sort(function() {
+    return 0.5 - Math.random();
+  });
+  arrayOfElements.forEach((element, index) => {
+    if (index >= arrayOfBackGrnds.length) {
+      element.style.backgroundPosition =
+        arrayOfBackGrnds[index - arrayOfBackGrnds.length];
+    } else {
+      element.style.backgroundPosition = arrayOfBackGrnds[index];
+    }
+  });
+}
+let arrayOfFlipCards = [];
+let boardLock = false;
+let countForMatches = 0;
+const firstOpenedCard = 0;
+const secondOpenedCard = 1;
+
+container.addEventListener("click", flip);
+
+function flip(event) {
+  let target = event.target;
```

Comment:
You can use destruction `fucntion flip({ target })`

Reply:
Done.